glitchdotcom/Glitch-Community
# Noindex tags and sitemap info

## Links
* https://app.clubhouse.io/glitch/story/5771/adding-a-noindex-meta-tag-removing-from-xml-sitemap-for-particular-projects

## GIF/Screenshots:

## Changes:
* If a user, team, or collection page doesn't have any projects on it, add a `noindex` tag
* Some basic info about our sitemaps (the part of this ticket that removes pages from the sitemap is done in a different remix)

## How To Test:
* Check that your empty teams and collections have a noindex meta tag in the header when you go to the page

## Feedback I'm looking for:
What can we add/clarify about the sitemaps (with the caveat that I'm still updating them for the new functionality)? @taravancil that's specifically why you're tagged 😀
